pkg/controller/generic/qtransform/options.go

```diff
@@ -68,11 +68,15 @@ type ControllerOptions struct {
 // ControllerOption is an option for QTransformController.
 type ControllerOption func(*ControllerOptions)
 
-// WithExtraMappedInput adds an extra  mapped inputs to the controller.
-func WithExtraMappedInput[I generic.ResourceWithRD](mapFunc MapperFuncGeneric[I]) ControllerOption {
+// WithExtraInput adds an extra input with the given kind to the controller.
+func WithExtraInput[I generic.ResourceWithRD](mapFunc MapperFuncGeneric[I], inputKind controller.InputKind) ControllerOption {
```

Comment:
idk, probably WithExtraMappedInputKind (?) as it's still Mapped, but custom kind

Reply:
done